src/docs/intro.html

```diff
@@ -119,6 +120,7 @@ <h1>Introduction</h1>
                       <a class="nav-link" href="#welcome">Welcome</a>
                       <a class="nav-link" href="#contributing">Contributing</a>
                       <a class="nav-link" href="#community">Community</a>
+                      <a class="nav-link" href="#nontechnical">Non technical explanation of Plasma</a>
```

Comment:
Not a big deal, but perhaps just: "Non-technical Explanation"
Leaving the words capitalized as it's a subject and omitting "Plasma" as the context of the website is Plasma.